src/main/java/com/sideteam/groupsaver/domain/club_schedule/domain/ClubSchedule.java

```diff
@@ -0,0 +1,58 @@
+package com.sideteam.groupsaver.domain.club_schedule.domain;
+
+import com.sideteam.groupsaver.domain.club_schedule.dto.ClubScheduleRequestDto;
+import com.sideteam.groupsaver.domain.common.BaseTimeEntity;
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import java.time.LocalDateTime;
+
+@SQLDelete(sql = "UPDATE ClubSchedule SET is_deleted = true WHERE id = ?")
+@Where(clause = "is_deleted = false")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Getter
+@Entity
+public class ClubSchedule extends BaseTimeEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    private boolean isDeleted;
+
+    @JoinColumn(nullable = false)
+    private Club club;
+
+    @Column(nullable = false)
+    private LocalDateTime meetAt;
+
+    @Column(nullable = false)
+    private String title;
+
+    // TODO 모임 일정 참석 최대 인원수?
+
+    public static ClubSchedule of(Club club, ClubScheduleRequestDto clubScheduleRequestDto) {
+        return new ClubSchedule(club, clubScheduleRequestDto.meetAt(), clubScheduleRequestDto.title());
+    }
+
+    public void update(LocalDateTime meetAt, String title) {
+        if (meetAt != null) {
+            this.meetAt = meetAt;
+        }
+        if (title != null) {
+            this.title = title;
+        }
+    }
```

Comment:
null 검사가 많지 않아서 if문으로 검사해도 괜찮다는 생각이드는데, 다르게 바꿔보고 싶다면 아래 코드처럼 변경하는건 어떠신가요??

```suggestion
    public void update(LocalDateTime meetAt, String title) {
        this.meetAt = Objects.requireNonNullElse(meetAt, this.meetAt);
        this.title = Objects.requireNonNullElse(title, this.title);
    }
```

Reply:
오우 if문을 깔끔하게 없앨 수 있어서 좋은 거 같습니다. 아이디어 감사합니다!